gxawsserverless/src/main/java/com/genexus/cloud/serverless/model/EventMessageSourceType.java

```diff
@@ -0,0 +1,8 @@
+package com.genexus.cloud.serverless.model;
+
+public class EventMessageSourceType {
+	public static String QueueMessage = "QueueMessage";
+	public static String ServiceBusMessage = "ServiceBusMessage";
+	public static String Timer = "Timer";
+	public static String StreamMessage = "StreamMessage";
+}
```

Comment:
Are this constants?
In that case it must contains final keyword ant theier names mus be Uppercase with _ for separate words